src/latexify/transformers/prefix_trimmer.py

```diff
@@ -0,0 +1,43 @@
+"""Transformer to trim package prefixes."""
+
+from __future__ import annotations
+
+import ast
+
+
+class PrefixTrimmer(ast.NodeTransformer):
+    """NodeTransformer to trim package prefixes.
```

Comment:
This transformer trims not only packages, but also any identifiers with the attribute syntax.

```suggestion
    """NodeTransformer to trim prefixes of identifiers.
```